eng/common/templates/job/execute-sdl.yml

```diff
@@ -11,7 +11,7 @@ parameters:
                                                                # Usage:
```

Comment:
All the changes under /eng/common/* should be reverted, these are files that live in arcade.